exo-go/src/terraform_file_helpers/index.go

```diff
@@ -94,9 +95,13 @@ func generateServiceModule(serviceName string, serviceConfig types.ServiceConfig
 }
 
 func generateDependencyModules(appConfig types.AppConfig) (string, error) {
+	appDir, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
```

Comment:
Please pass in the cwd instead of grabbing it inside this function. That will make this easier to test and help us limit the calls to `os.Getwd()`.